demo/src/pages/layout.js

```diff
@@ -0,0 +1,188 @@
+                  Vivamus a egestas nulla, nec eleifend dolor. Praesent risus
+                  felis, maximus vitae nunc id, aliquam varius mauris. Ut
+                  aliquet sagittis eros ac sodales. In faucibus justo vitae
+                  nulla pellentesque pulvinar. Praesent nec diam eu turpis
+                  consectetur rhoncus. Mauris a tincidunt orci. Nam consectetur
+                  erat non sagittis eleifend. Aenean id auctor sapien.
+                </p>
+              </Tabs.Page>
+            </Tabs.Pages>
+          </Tabs>
+        </PropsDisplay>
+        <PropsDisplay componentInfo={tableData}>
+          <Table>
+            <thead>
+              <tr>
+                <td>Col 1</td>
+                <td>Col 2</td>
+                <td>Col 3</td>
+              </tr>
+            </thead>
+            <tbody>
+              {Array(5)
+                .fill()
+                .map((item, i) => (
+                  <tr>
+                    <td>Item {i * 3 + 1}</td>
+                    <td>Item {i * 3 + 2}</td>
+                    <td>Item {i * 3 + 3}</td>
+                  </tr>
+                ))}
+            </tbody>
+          </Table>
+        </PropsDisplay>
+        <PropsDisplay componentInfo={sideData}>
+          <Side>
+            A column of content, to be passed as a child to SideBySide.
+          </Side>
+        </PropsDisplay>
+        <PropsDisplay componentInfo={sideBySideData}>
+          <SideBySide>
+            <Side>
+              <h4> This Side contains a code block. </h4>
+              <CodeBlock language="json">{jsonExample}</CodeBlock>
+            </Side>
+            <Side>
+              <h4>And this Side contains text.</h4>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+                pulvinar ante eget purus porttitor suscipit. Donec a eros eget
+                ante elementum sollicitudin.
+              </p>
+              <p>
+                Quisque ut dui posuere, faucibus enim eu, convallis eros.
+                Curabitur sodales sapien eu lorem dapibus, sed ullamcorper sem
+                faucibus. Nullam bibendum risus lacinia erat congue, at
+                efficitur quam tempus.
+              </p>
+            </Side>
+          </SideBySide>
+        </PropsDisplay>
+      </Layout.Content>
+    </Layout.Main>
+  );
+};
+
+export const pageQuery = graphql`
+  query LayoutQuery {
+    allJson(
+      filter: {
+        displayName: {
+          in: [
+            "Collapser"
+            "CollapserGroup"
+            "Tabs"
+            "Table"
+            "SideBySide"
+            "Side"
+          ]
+        }
+      }
+    ) {
+      edges {
+        node {
+          description
+          displayName
+          props {
+            description
+            name
+            required
+            type
+          }
+        }
+      }
+    }
+  }
+`;
+
```

Comment:
Sorry to nitpick, but could we add a PropTypes field? My linter yells at me if we don't.
```suggestion

LayoutPage.propTypes = {
  data: PropTypes.object.isRequired,
};

```

Reply:
This has been updated!